data_api/snapshot_scheduler/src/test_snapshot_scheduler.py

```diff
@@ -15,10 +15,20 @@ def utcnow(cls):
 
 @mock.patch('datetime.datetime', patched_datetime)
 def test_writes_message_to_sqs(sns_client, topic_arn):
-    snapshot_scheduler.main(sns_client=sns_client)
+    target_bucket_name="target_bucket_name"
+    es_index="es_index"
+
+    snapshot_scheduler._run(
```

Comment:
Ick. Underscore methods are generally treated as “private” – you can access them, but they’re treated as internal and you shouldn’t rely on them. Better would be to set up the environment variables you expect, and call the `main` function.